packages/blade/src/components/BottomSheet/BottomSheetFooter.native.tsx

```diff
@@ -23,20 +20,7 @@ const _BottomSheetFooter = ({
       touchAction="none"
       zIndex={2}
     >
```

Comment:
Not making this outer component part of BaseFooter? also intentional to set backgroundColor on this outer component? BaseFooter will set the color no?

Reply:
> Not making this outer component part of BaseFooter

No because these outer ones are very platform specific wrappers, for example in Web we spread gesture handlers and add special CSS to prevent touch action and in native we don't put much stuff.

Yes backgroundColor is expected, otherwise on RN if scroll bounces to bottom the body content will show up from behind.